task/buildah/0.1/buildah.yaml

```
@@ -207,6 +213,13 @@ spec:
       [ -n "$COMMIT_SHA" ] && LABELS+=("--label" "vcs-ref=$COMMIT_SHA")
       [ -n "$IMAGE_EXPIRES_AFTER" ] && LABELS+=("--label" "quay.expires-after=$IMAGE_EXPIRES_AFTER")
 
+      ENTITLEMENT_PATH="/entitlement"
+      if [ -n "${ENTITLEMENT_SECRET}" ] && [ -d "$ENTITLEMENT_PATH" ]; then
+        cp -r /entitlement /tmp/entitlement
```

```suggestion
        cp -r "$ENTITLEMENT_PATH" /tmp/entitlement
```
just for consistency

---

updated